### PR TITLE
Stylesheet fixes

### PR DESCRIPTION
**Pull request by MisterMakerNL**
I'll add all the fixes that users come across here.

### CODE REVIEW

**Comment by MisterMakerNL** [comment]
Issue thread:
https://github.com/FreeCAD/FreeCAD/issues/14745

**Comment by MisterMakerNL** [comment]
Radio buttons reported by Kadet should be fixed now, I boosted all icons sizes since I saw artifacts on lower resolution.
Only the Qsint icons stay the same. 
Should look like this now:
image

image

**Comment by MisterMakerNL** [comment]
I am a bit bussy atm with life and such. 
I updated 2 overlay stylesheets but I still have too update the others 2.

**Comment by MisterMakerNL** [comment]
I marked this as ready for review.
Most of it are bugs fixes.
Only the naming a UI change.
Also don't want to do to many bug fixes in one PR, so it is probably wise to push.

**Comment by MisterMakerNL** [comment]
Please note that this wil change default no-stylesheet to Qstyle: Fusion.
I know @kadet1090 didn't like it, but my reasoning is that it make the Ui more stable and we have less things to worry about.
As Kadet pointed out this will most like impact mac users the most. But since our development on UI for mac isn't the most active, I think this is a good thing.
The idea is too test it out and decide before 1.0 release if we keep this in.

So too anyone reading this in the future and thinks what idiot did this, it is me and it's a test, that's what development version are for your welcome. :P

**Comment by adrianinsaval** [comment]
can you please split the change to Fusion by default into a separate PR? we want to merge all this as a single commit but we want that specifically in a separate commit since it's a controversial change that we might need to revert

**Comment by MisterMakerNL** [comment]
> can you please split the change to Fusion by default into a separate PR? we want to merge all this as a single commit but we want that specifically in a separate commit since it's a controversial change that we might need to revert

okidoki

**Comment by adrianinsaval** [comment]
why are translation files being touched here?

**Comment by MisterMakerNL** [comment]
> why are translation files being touched here?

Themes are renamed too "Freecad dark" "Freecad light" and "Freecad classic". And startpage is using translation files for these names.

**Comment by chennes** [comment]
You should never edit TS files directly: those are generated via CrowdIn (our translation tool). Any manual changes here will be lost.

**Comment by MisterMakerNL** [comment]
> You should never edit TS files directly: those are generated via CrowdIn (our translation tool). Any manual changes here will be lost.

Oh no. Then I'll remove them.

**Comment by MisterMakerNL** [comment]
Ok I removed them could someone do the translation for me?
IT's just adding Freecad before the theme name:
Freecad Dark
Freecad Light
Freecad Classic

**Comment by furgo16** [comment]
> Ok I removed them could someone do the translation for me?

Generally the way it works is that when new strings are added to the code and the PR is merged, the newly translatable messages will be uploaded to Crowdin. There, translators will see and translate them. Afterwards, the new translations will be downloaded and merged in a new PR. The latter step seems to be happening manually at more or less regular intervals.

In summary, once this PR is merged, all translation steps will be done for you.

**Comment by MisterMakerNL** [comment]
image
new file buttons settings added to preference:
image

**Comment by adrianinsaval** [comment]
> New file buttons settings added to preference: image

I'm confused is this how the startpage will look like after this PR?

**Comment by MisterMakerNL** [comment]
> > New file buttons settings added to preference: image
> 
> I'm confused is this how the startpage will look like after this PR?

The green color yes, the sizing issue no, ui that is another  issue that needs to be solved.
It's green because it's only accept single color input and the normal button have gradients. 
So to not make it look like a text window I though I give it a special color. I also kind did it for testing and liked it.

**Comment by adrianinsaval** [comment]
> > > New file buttons settings added to preference: image
> > 
> > 
> > I'm confused is this how the startpage will look like after this PR?
> 
> The green color yes, the sizing issue no, that is not caused by this PR this is another issue that needs to be solved and I can't I tried, but I don't understand the code. It's green because it's only accept single color input and the normal button have gradients. So to not make it look like a text window I though I give it a special color. I also kind did it for testing and liked it.

I don't think I can merge such a big visual change at this point. We can consider this for post 1.0 but if we still want to merge this for 1.0 we have to take that out.

**Comment by MisterMakerNL** [comment]
I also changed default colors to green, just to work around the bug.
If someone got some time to burn on fixing it better then please do!
image

**Comment by MisterMakerNL** [comment]
> > The green color yes, the sizing issue no, that is not caused by this PR this is another issue that needs to be solved and I can't I tried, but I don't understand the code. It's green because it's only accept single color input and the normal button have gradients. So to not make it look like a text window I though I give it a special color. I also kind did it for testing and liked it.
> 
> I don't think I can merge such a big visual change at this point. We can consider this for post 1.0 but if we still want to merge this for 1.0 we have to take that out.

Unless someone can fix up the startpage so you can switch theme's without breaking the UI then it would be better too keep it green. Then at least it won't break the program. This is a bug fix I am not doing it because I like it to be green.

**Comment by MisterMakerNL** [comment]
Please merge this it's not perfect but the best way too move forward is by merging it and testing it in the weekly.

**Comment by MisterMakerNL** [comment]
@wwmayer Coudl you help here, the stat page seems to check for a stylesheet but it seems not to work.
It just ignores stylesheet all together!?

**Comment by MisterMakerNL** [comment]
Kadet already seems to have find a fix will try to push this this evening. Which is in about 10 hours.

**Comment by MisterMakerNL** [comment]
I reverted the green color it now uses the stylesheet default, for classic this is the default button of the OS or the Qstyle.
The 2 others use the button colors.
image
image
image

**Comment by MisterMakerNL** [comment]
I have added https://github.com/FreeCAD/FreeCAD/pull/15328

File: src/Mod/Start/Gui/ThemeSelectorWidget.cpp [hunk 1]
```diff
@@ -156,7 +156,7 @@ void ThemeSelectorWidget::retranslateUi()
     _titleLabel->setText(QLatin1String("<h2>") + tr("Theme") + QLatin1String("</h2>"));
     _descriptionLabel->setText(tr("Looking for more themes? You can obtain them using "
                                   "<a href=\"freecad:Std_AddonMgr\">Addon Manager</a>."));
-    _buttons[static_cast<int>(Theme::Dark)]->setText(tr("Dark theme", "Visual theme name"));
-    _buttons[static_cast<int>(Theme::Light)]->setText(tr("Light theme", "Visual theme name"));
-    _buttons[static_cast<int>(Theme::Classic)]->setText(tr("Classic", "Visual theme name"));
+    _buttons[static_cast<int>(Theme::Dark)]->setText(tr("Freecad Dark", "Visual theme name"));
```
Comment (maxwxyz) [inline src/Mod/Start/Gui/ThemeSelectorWidget.cpp:159]: "FreeCAD" would be better than "Freecad", also in the following two lines

File: src/Mod/Start/Gui/ThemeSelectorWidget.cpp [hunk 2]
```diff
@@ -132,13 +132,13 @@ void ThemeSelectorWidget::themeChanged(Theme newTheme)
     auto prefPackManager = Gui::Application::Instance->prefPackManager();
     switch (newTheme) {
         case Theme::Classic:
-            prefPackManager->apply("Classic");
+            prefPackManager->apply("Freecad Classic");
```
Comment (maxwxyz) [inline src/Mod/Start/Gui/ThemeSelectorWidget.cpp:135]: not sure if the typo has to be corrected here too

File: src/Mod/Start/Gui/ThemeSelectorWidget.cpp [hunk 3]
```diff
@@ -77,12 +77,12 @@ void ThemeSelectorWidget::setupButtons(QBoxLayout* layout)
...
         }
         else if (theme.first == Theme::Light
-                 && styleSheetName.contains(QLatin1String("Light theme"),
+                 && styleSheetName.contains(QLatin1String("Freecad Light"),
                                             Qt::CaseSensitivity::CaseInsensitive)) {
             button->setChecked(true);
         }
         else if (theme.first == Theme::Dark
-                 && styleSheetName.contains(QLatin1String("Dark theme"),
+                 && styleSheetName.contains(QLatin1String("Freecad ark"),
```
Comment (maxwxyz) [inline src/Mod/Start/Gui/ThemeSelectorWidget.cpp:85]: "ark" typo.
Also FreeCAD if possible

File: src/Mod/Start/Gui/ThemeSelectorWidget.cpp [hunk 4]
```diff
@@ -77,12 +77,12 @@ void ThemeSelectorWidget::setupButtons(QBoxLayout* layout)
             button->setChecked(true);
         }
         else if (theme.first == Theme::Light
-                 && styleSheetName.contains(QLatin1String("Light theme"),
+                 && styleSheetName.contains(QLatin1String("Freecad Light"),
```
Comment (maxwxyz) [inline src/Mod/Start/Gui/ThemeSelectorWidget.cpp:80]: FreeCAD

File: src/Mod/Start/Gui/ThemeSelectorWidget.cpp [hunk 5]
```diff
@@ -55,9 +55,9 @@ void ThemeSelectorWidget::setupButtons(QBoxLayout* layout)
     if (!layout) {
         return;
     }
-    std::map<Theme, QString> themeMap {{Theme::Classic, tr("Classic")},
-                                       {Theme::Dark, tr("Dark theme")},
-                                       {Theme::Light, tr("Light theme")}};
+    std::map<Theme, QString> themeMap {{Theme::Classic, tr("Freecad Classic")},
```
Comment (maxwxyz) [inline src/Mod/Start/Gui/ThemeSelectorWidget.cpp:58]: "FreeCAD" would be better than "Freecad", also in the following two lines

File: src/Gui/PreferencePacks/package.xml [hunk 6]
```diff
@@ -9,7 +9,7 @@
 
   <content>
     <preferencepack>
-      <name>Classic</name>
+      <name>Freecad Classic</name>
```
Comment (maxwxyz) [inline src/Gui/PreferencePacks/package.xml:12]: "FreeCAD" would be better than "Freecad"

File: src/Gui/PreferencePacks/package.xml [hunk 7]
```diff
@@ -19,22 +19,22 @@
       <tag>classic theme</tag>
     </preferencepack>
 	<preferencepack>
-	  <name>Light theme</name>
+	  <name>Freecad Light</name>
```
Comment (maxwxyz) [inline src/Gui/PreferencePacks/package.xml:22]: "FreeCAD" would be better than "Freecad"

File: src/Gui/PreferencePacks/package.xml [hunk 8]
```diff
@@ -19,22 +19,22 @@
...
-      <version>0.1.0</version>
+      <version>1.0.0</version>
       <tag>built-in</tag>
       <tag>background</tag>
-	  <tag>light</tag>
+	    <tag>light</tag>
     </preferencepack>
 	<preferencepack>
-	  <name>Dark theme</name>
+	  <name>Freecad Dark</name>
```
Comment (maxwxyz) [inline src/Gui/PreferencePacks/package.xml:31]: "FreeCAD" would be better than "Freecad"

**Review by maxwxyz — CHANGES_REQUESTED** [review]
Small typo, also "FreeCAD" would be better than "Freecad" IMO as it is spelled like this everywhere.

File: src/Gui/PreferencePacks/package.xml (hunk 6 above)
Comment (MisterMakerNL) [reply src/Gui/PreferencePacks/package.xml]: image
image

**Review by maxwxyz — APPROVED** [review]

**Comment by Syres916** [comment]
If you change `Freecad` to `FreeCAD` in the code then all the folder structure and names of the Preference Packs have to match hence the two Ubuntu test builds have failed.

**Comment by MisterMakerNL** [comment]
Github desktop doesn't pick up changed in caps....

**Comment by Syres916** [comment]
Yep it's a known bug with Github Desktop for about 5 years. All I can suggest is with max's approval you back out the code changes back to `Freecad` on the agreement that after merging of this PR you'll submit a further PR through the Github _website_ to correct all the `Freecad` 's.

**Comment by MisterMakerNL** [comment]
I deleted all files and directories then I committed that, then I undo  that in file browser. Then I commited to undo folders/files.
Now they are renamed.
Kind of ugly but it seems to work.

**Comment by MisterMakerNL** [comment]
I think I got it all!

**Comment by MisterMakerNL** [comment]
All checks passed so I am not touching this PR until it hopefully gets merges tomorrow.

**Comment by chennes** [comment]
We are trying to avoid any translation string changes that don't correct actual errors: can you please split those out into a second PR that we'll mark for the post-1.0 milestone (and also fix those conflicts)?

**Comment by kadet1090** [comment]
IMHO this should be changed now or never and it should definitively be changed.

**Comment by chennes** [comment]
OK -- we discussed this in the review meeting and decided to go ahead with the string changes. So once it's rebased we will merge.

**Comment by MisterMakerNL** [comment]
You cannot halt this PR because the name has to change to FreeCAD then halt it again because it is changed FreeCAD. ;)

**Comment by MisterMakerNL** [comment]
Should be done now.

**Pull request merged**